contracts/HubPool.sol

```diff
@@ -513,7 +557,7 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         bytes32 poolRebalanceRoot,
         bytes32 relayerRefundRoot,
         bytes32 slowRelayRoot
-    ) public override nonReentrant noActiveRequests {
+    ) public override nonReentrant noActiveRequests unpaused {
```

Comment:
Should we be adding the `unpaused` modifier to other non-admin functions like `addLiquidity`, `removeLiquidity` and more subtle state-modifying methods like `sync` and `claimProtocolFeesCaptured`?

Also, what about `disputeRootBundle`?

Reply:
I think the intention is to _only_ block proposals from moving forward, right? Do we want to block any normal LP or fee functionality?

Reply:
I agree with this. We also have other ways to pause liquidity methods via `enableL1TokenForLiquidityProvision`

Reply:
I agree this is the only function that should have this.